tests/php/BlockExtendPluginTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Tests for class BlockExtendPlugin.
+ */
+
+namespace XWP\BlockExtendTest;
+
+use WP_Mock;
+use Mockery;
+use XWP\BlockExtend\BlockExtendPlugin;
+use XWP\BlockExtend\Plugin;
+
+/**
+ * Tests for class BlockExtendPlugin.
+ */
+class BlockExtendPluginTest extends BlockExtendTestCase {
+
+	/**
+	 * Test init.
+	 *
+	 * @covers XWP\BlockExtend\BlockExtendPlugin::init()
+	 */
+	public function test_init() {
+		$plugin = new BlockExtendPlugin( Mockery::mock( Plugin::class ) );
+
+		WP_Mock::expectActionAdded( 'enqueue_block_editor_assets', [ $plugin, 'enqueue_editor_assets' ], 10, 1 );
```

Comment:
@kienstra This now only deals with the responsibilities of this plugin -- registering the hook with the right arguments. It assumes that WP core API does what it is supposed to do.

Reply:
Good idea. Not testing the internals of Core.

Reply:
Ah, good idea.